src/store/getters.ts

```diff
@@ -48,8 +48,7 @@ export default {
   },
 
   /** Is True if there is a blocker including firm compliance. */
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  hasBlocker (state: StateIF, getters: any): boolean {
+  hasBlocker (_state: StateIF, getters: any): boolean {
```

Comment:
A little syntax thing: if you name your formal parameters starting with a `_`, eslint ignores them.